### ISSUE TITLE
Economy commands?

### ISSUE THREAD

**Issue by Shinwulf** [suvanl/delet3#24]
Hey Suvanl, I find your work on delet 3 pretty decent! 

Would you add an economy system to it? Like, an item shop that changes every week, daily rewards with streak and more. It would be cool to see that! If possible, I would love to help you with that!

**Comment by suvanl** [comment]
Hi, thank you!

An economy system as per your suggestion sounds like a really fun addition. 

A basic points system already exists, where users get a random number of points (between 10 and 20) whenever they send a message within a specified cooldown period. So, we could use these points as the currency for the item shop in the economy system. 

Do you happen to have any suggestions on how this could be implemented? Any help would be much appreciated, of course! :)

**Comment by Shinwulf** [comment]
Sure, if you want to we can talk about it on Discord, my username is Shin#0484. (:

**Comment by NotSure70** [comment]
actually, I have an idea for thing if I can share :)

**Comment by suvanl** [comment]
Of course! Feel free to outline your idea(s) here or let me know if you'd rather discuss it on Discord.

**Comment by NotSure70** [comment]
lets discuss on discord. you can add me @ DVG#6945
